Services/Database/interfaces/interface.ilTableLockInterface.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+/**
+ * Class ilTableLockInterface
+ *
+ * @author Fabian Schmid <fs@studer-raimann.ch>
+ */
+interface ilTableLockInterface {
+
+	/**
+	 * @param bool $lock_bool
```

Comment:
Please add a sentence about the semantics here. There would be no clue how to use or implement that interface otherwise.